src/pages/resultsView/mutation/Mutations.tsx

```diff
@@ -28,12 +30,19 @@ export default class Mutations extends React.Component<
 > {
     private userSelectionStore: MutationMapperUserSelectionStore;
 
-    @observable mutationsGeneTab: string;
+    @observable public selectedGeneSymbol: string = this.props.store.genes
+        .result![0].hugoGeneSymbol;
```

Comment:
let's do this in the constructor if possible. I think it is okay to assume that result may be undefined, so here:
```suggestion
    @observable public selectedGeneSymbol: string | undefined;
```

in the constructor:
```suggestion
    this.selectedGeneSymbol = props.store.genes.result ? props.store.genes.result[0].hugoGeneSymbol: undefined;
```